botorch/optim/initializers.py

```diff
@@ -226,6 +229,130 @@ def gen_one_shot_kg_initial_conditions(
     return ics
 
 
+def gen_value_function_initial_conditions(
+    acq_function: Union[PosteriorMean, qSimpleRegret],
```

Comment:
We can be more general here (see comment below)

```suggestion
    acq_function: AcquisitionFunction,
```